### PR TITLE
Update Weekly-Meetings.md

### PR DESCRIPTION
**Pull request by sparkler0323**

### CODE REVIEW

**Comment by KirstieJane** [comment]
Hi @sparkler0323 - please upload your abstracts & bios file with justification to attend Cambridge Networks Day by end of day on Tuesday so I can book tickets.

File: Ang-Li/abstracts_bios.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+### Cambridge Networks Day
+
+**Website**: https://www.cnn.group.cam.ac.uk/cambridge-networks-day
+
+**Date**: 29 August 2019
+
+**Application**: 
+I hope to attend the Cambridge network day, as I'm super interested at network science.
+As a Ph.D. student focusing on neuroimaging and data science, I already have some background knowledge of network and its application 
+in neuroimaging. Our brain, just like the other complex systems in the real world, is a small world network, and the topological structure
```
Comment (KirstieJane) [inline Ang-Li/abstracts_bios.md]: I know I'm really annoying about this - please can you put each new sentence on a new line?

File: Ang-Li/abstracts_bios.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+**Application**: 
+I hope to attend the Cambridge network day, as I'm super interested at network science.
+As a Ph.D. student focusing on neuroimaging and data science, I already have some background knowledge of network and its application 
+in neuroimaging. Our brain, just like the other complex systems in the real world, is a small world network, and the topological structure 
+of brain is abnormal in many brain disorders. 
+Although we have deeper understanding of our brain using the framework of network/graph theory, we have 
+very limited applications to help the real world in neuroimaging. 
+Thus, it's a great opportunity for me to know more network applications in diverse fields, and to learn novel methods from the 
+differernt fields.
```
Comment (KirstieJane) [inline Ang-Li/abstracts_bios.md]:
```suggestion
different fields.
```

File: Ang-Li/abstracts_bios.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+I am particularly looking forward to listening the talk from Prof. Danielle Bassett, whose lab has 
+developed some useful tools from network science to reslove scientific problems including brain science.
```
Comment (KirstieJane) [inline Ang-Li/abstracts_bios.md]:
```suggestion
developed some useful tools from network science to resolve scientific problems including brain science. 
```

File: Ang-Li/abstracts_bios.md (hunk 1 above)
Comment (sparkler0323) [reply Ang-Li/abstracts_bios.md]: Of course, have fixed this point

Comment (KirstieJane) [reply Ang-Li/abstracts_bios.md]: Not all of the sentences are on new lines. Please read through carefully and fix all of them in this paragraph.

File: Ang-Li/abstracts_bios.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+**Date**: 29 August 2019
+
+**Application**: 
+I hope to attend the Cambridge network day, as I'm super interested at network science.
+As a Ph.D. student focusing on neuroimaging and data science, I already have some background knowledge of network and its application in neuroimaging. 
+Our brain, just like the other complex systems in the real world, is a small world network, and the topological structure of brain is abnormal in many brain disorders. 
+Although we have deeper understanding of our brain using the framework of network/graph theory, we have very limited applications to help the real world in neuroimaging. 
+Thus, it's a great opportunity for me to know more network applications in diverse fields, and to learn novel methods from the 
+different fields.
```
Comment (KirstieJane) [inline Ang-Li/abstracts_bios.md]: Should be on line above.

**Comment by KirstieJane** [comment]
This has one minor requested change and then can be merged.

**Comment by KirstieJane** [comment]
Pinging @sparkler0323 to address the comment and merge conflict.

File: Ang-Li/abstracts_bios.md (hunk 4 above)
Comment (sparkler0323) [reply Ang-Li/abstracts_bios.md]: Have fixed that 😸

**Comment by KirstieJane** [comment]
@sparkler0323 - do you know how to fix the merge conflict? Let me know if you need any help.

**Comment by sparkler0323** [comment]
Thanks Kirstie, it seems worked now~

**Pull request merged**